.eslintrc.cjs

```diff
@@ -0,0 +1,62 @@
+module.exports = {
```

Comment:
Out of curiosity, why is this a separate file and no longer in the root `package.json`? Is that what allows the removal of all the dummy `"extends"` in the sub `package.json`s?